nncf/experimental/post_training/factories.py

```diff
@@ -0,0 +1,91 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from build.lib.nncf.experimental.onnx.graph.metatypes.onnx_metatypes import LAYERS_WITH_BIAS_METATYPES
+from build.lib.nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNX_OPERATION_METATYPES
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.common.tensor import NNCFTensor
+from nncf.common.tensor_statistics.collectors import MeanStatisticCollector
+from nncf.common.utils.backend import BackendType
+from nncf.common.utils.backend import infer_backend_from_model
+from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
+from nncf.experimental.onnx.graph.transformations.commands import ONNXBiasCorrectionCommand
+from nncf.experimental.onnx.graph.transformations.commands import ONNXModelExtractionCommand
+from nncf.experimental.onnx.graph.transformations.commands import ONNXTargetPoint
+from nncf.experimental.onnx.statistics.collectors import ONNXMeanStatisticCollector
+from nncf.experimental.onnx.tensor import ONNXNNCFTensor
+
+
+class NNCFGraphFactory:
```

Comment:
It looks like we need to combine graph related factories into the one single and instantiate only one object responsible for a model transformations.